lib/datasource/local/settings_storage.dart

```diff
@@ -205,6 +205,7 @@ class _SettingsStorage {
   Future<void> saveAccount(String accountName, String privateKey) async {
     _accountName = accountName;
     _privateKey = privateKey;
```

Comment:
what is this doing then?

Reply:
its used to keep the current private key and therefore by the public getter passcode.

Reply:
_privateKey = privateKey;

This line of code i mean

Reply:
Yes it's what i said  
String get passcode => _passcode;